public/index.html

```diff
@@ -1,27 +1,110 @@
-	<script defer src="https://www.gstatic.com/firebasejs/7.17.1/firebase-analytics.js"></script>
-	<script defer src="https://www.gstatic.com/firebasejs/7.17.1/firebase-firestore.js"></script>
-	<script src="https://kit.fontawesome.com/0d4aacb00e.js" crossorigin="anonymous"></script>
-	<title>MyCovidConnect: Find Nearest Healthcare facility</title>
-</head>
-
-<body>
-	<noscript>You need to enable JavaScript to run this app.</noscript>
-	<div id="root"></div>
-</body>
-
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link type="text/plain" rel="author" href="humans.txt" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-app.js"></script>
+    <script
+      defer
+      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-analytics.js"
+    ></script>
+    <script
+      defer
+      src="https://www.gstatic.com/firebasejs/7.17.1/firebase-firestore.js"
+    ></script>
+    <script
+      src="https://kit.fontawesome.com/0d4aacb00e.js"
+      crossorigin="anonymous"
+    ></script>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="manifest" href="manifest.json" />
+    <title>MyCovidConnect: Find Nearest Healthcare facility</title>
+    <style>
+      .add-button {
+        position: absolute;
+        top: 1px;
+        left: 1px;
+      }
+      .add-to-btn {
+        background-color: #f15c08;
+        padding: 10px;
+        border: 0px;
+        color: wheat;
+        border-radius: 20px;
+        font-weight: normal;
+        text-transform: capitalize;
+        font-family: "Lato", sans-serif;
+      }
+    </style>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <div class="add-to">
+      <div style="position: fixed; bottom: 10px; right: 10px; color: green">
+        <button class="add-to-btn">Install Our App</button>
+      </div>
+    </div>
+    <script>
+      if ("serviceWorker" in navigator) {
+        window.addEventListener("load", function () {
+          navigator.serviceWorker.register("/serviceworker.js").then(
+            function (registration) {
```

Comment:
enable this only if the DEBUG variable is set to true. you can import debug variable from constants.js;
If there isn't one, then create a variable as such. follow the pattern there.